README.md

```diff
@@ -68,6 +79,6 @@ UserApp.add_all_users = function(data) {
   console.log(data);
 };
 ```
-
+I mean, a quick fix is to just pass 'data' in the console.log where it says success, or add another log for the data, if what you're trying to do is display the data. Otherwise it depends on what you want to do, which I can't say is entirely clear. UserApp.all_all_users isn't part of the .done function, for example, so if you wanted to add it you'd have to put it in there.
```

Comment:
Yup, you pretty much got this. `UserApp.add_all_users(data);` needs to be inside of the `.done()` callback in order to have access to "data".